src-tauri/src/northstar/mod.rs

```diff
@@ -1,6 +1,12 @@
 //! This module deals with handling things around Northstar such as
 //! - getting version number
 
+pub const CORE_MODS: [&str; 3] = [
```

Comment:
I suppose `3` is the length of the array?

Reply:
Yup. Global consts require their type to be explicitly defined in Rust. So basically `str` array of size 3 ^^